Content.Server/Explosion/Components/TriggerOnProximityComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Physics.Collision.Shapes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using System;
+
+namespace Content.Server.Explosion.Components
+{
+    [RegisterComponent]
+    public class TriggerOnProximityComponent : Component
+    {
+        public override string Name => "TriggerOnProximity";
+        public TimeSpan LastTrigger = TimeSpan.Zero;
+        public string ProximityFixture { get; } = "proximity-fixture";
+
+        [DataField("shape", required: true)]
+        public IPhysShape Shape { get; set; } = default!;
+
+
+        [DataField("enabled")]
+        public bool enabled;
+        
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool Enabled
+        {
+            set
+            {
+                EntitySystem.Get<TriggerSystem>().SetProximityFixture(Owner.Uid, this, value);
+                enabled = value;
+            }
+            get => enabled;
+        }
```

Comment:
These names are stinky, just make the VV one EnabledVV like vera suggested and make the other Enabled. I would also make enabled default to true.